packages/@sanity/portable-text-editor/e2e-tests/__tests__/writingTogether.collaborative.test.ts

```diff
@@ -669,8 +669,8 @@ describe('collaborate editing', () => {
     `)
     const selectionA = await editorA.getSelection()
     expect(selectionA).toEqual({
-      anchor: {path: [{_key: 'A-8'}, 'children', {_key: 'A-7'}], offset: 0},
-      focus: {path: [{_key: 'A-8'}, 'children', {_key: 'A-7'}], offset: 0},
+      anchor: {path: [{_key: 'A-6'}, 'children', {_key: 'A-5'}], offset: 0},
```

Comment:
Not sure how the user A was in `{path: [{_key: 'A-8'}, 'children', {_key: 'A-7'}], offset: 0}` before given that block doesn't exists, and why now it is corrected. Any thoughts @skogsmaskin ?

Reply:
Updated the snapshot to reflect the change, snapshot compared was the previous value not the new one after the second press on the enter key